feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/AccountInteractorImpl.kt

```diff
@@ -190,4 +191,14 @@ class AccountInteractorImpl(
             }
         }
     }
+
+    override suspend fun switchToNotDeactivatedAccountIfNeeded() {
+        val metaAccount = accountRepository.getSelectedMetaAccount()
+        if (metaAccount.status == LightMetaAccount.Status.DEACTIVATED) {
+            val metaAccounts = getMetaAccounts()
+            if (metaAccounts.isNotEmpty()) {
+                accountRepository.selectMetaAccount(metaAccounts.first().id)
```

Comment:
It is safer to switch to first active meta account.

Reply:
Agree